server/utils/middleware/articleFilters.js

```diff
@@ -44,6 +44,14 @@ const articleFilters = (req, res, next) => {
       [Op.like]: `%${lastName || ''}%`,
     },
   };
+
+  if (tag) {
+    req.filterByTag = {
```

Comment:
You need to test for this, it is dropping the test coverage.

Reply:
done